src/proto/message_translation/tbots_geometry.cpp

```diff
@@ -102,9 +102,8 @@ Circle createCircle(const TbotsProto::Circle& circle)
 VelocityObstacle createVelocityObstacle(
     const TbotsProto::VelocityObstacle& velocity_obstacle_msg)
 {
-    VelocityObstacle velocity_obstacle;
-    velocity_obstacle.apex_  = createVector(velocity_obstacle_msg.apex());
-    velocity_obstacle.side1_ = createVector(velocity_obstacle_msg.left_side());
-    velocity_obstacle.side2_ = createVector(velocity_obstacle_msg.right_side());
-    return velocity_obstacle;
+    Vector apex_  = createVector(velocity_obstacle_msg.apex());
+    Vector side1_ = createVector(velocity_obstacle_msg.left_side());
+    Vector side2_ = createVector(velocity_obstacle_msg.right_side());
+    return VelocityObstacle(apex_, side1_, side2_);
```

Comment:
nit: the `side` variable names can be updated to left/right and they dont need an underscore anymore.

Reply:
updated